app/views/user_mailer/assignment_reminder.html.erb

```diff
@@ -1,7 +1,7 @@
 
 <p>Hello <%= @employee.first_name %>,  </p>
 
-<p>This is a reminder that you are assigned to kitchen duties this week. </p>
+<p>This is a reminder that you are assigned to kitchen duties from <%= print_date(@assignment.week) %>. </p>
```

Comment:
Why "from"? I might say "from x day to y day" but probably just "next week (week of January 1st)".